frontend/src/metabase/dashboard/components/DashboardActions.jsx

```diff
@@ -47,6 +47,7 @@ export const getDashboardActions = (
 
     buttons.push(
       <PopoverWithTrigger
+        ref="popover"
```

Comment:
This was no misplaced `ref`. I couldn't find it in use because it was referenced using `self` in a functional component. Oh wow!
See this commit for more info: https://github.com/metabase/metabase/pull/15427/commits/b3d64b8f7a982e0371d225d0d624ecbeaf776b76